pennylane_qiskit/qiskit_device.py

```diff
@@ -178,7 +196,14 @@ def reset(self):
         self._current_job = None
         self._state = None  # statevector of a simulator backend
 
-    def apply(self, operations, **kwargs):
+    def apply(self, operations, persist_transpile_options=False, **kwargs):
+        """This method applies the given set of operations on the qiskit ciruit.
+
+        Args:
+            operations (List[PennyLane.operation]): operations to be applied
+            persist_transpile_options (bool, optional): If true, transpile options
+            will be used for furture runs. Defaults to False.
+        """
```

Comment:
The `apply` method and its docstring is inherited, so its docstring doesn't need to be redefined.
```suggestion
```